TestAssets/TestProjects/StandaloneApp/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": "1.0.0-rc3-*",
+        "Newtonsoft.Json": "9.0.1-beta1"
+      }
+    }
+  },
+  "runtimes": {
+    "win7-x64": {},
+    "win7-x86": {},
+    "win8-x64": {},
+    "win8-x86": {},
+    "win81-x64": {},
+    "win81-x86": {},
```

Comment:
Issue was that I am calling dotnet build --runtime win10-x64 and without these rids it fails. I'm not sure what the workaround is here that still ensures I get rid specific output in --output

---
In reply to: [66346647](https://github.com/dotnet/core-setup/pull/97#discussion_r66346647) [](ancestors = 66346647)